src/quafel/pipelines/visualization/nodes.py

```diff
@@ -463,7 +444,8 @@ def qubits_time_viz(evaluations_combined: Dict, skip_frameworks: List):
 def shots_time_viz(evaluations_combined: Dict, skip_frameworks: List):
     figures = {}
 
-    # those two color sets are well suited as they correspond regarding their color value but differ from their luminosity and saturation values
+    # those two color sets are well suited as they correspond regarding
+    # their color value but differ from their luminosity and saturation values
     main_colors_it = iter(design.qual_main)
     sec_colors_it = iter(design.qual_second)
```

Comment:
You are using the color sets at different places, so you might consider having a function to get your current color set.

Reply:
the idea here is to have a class where the colors are specified and then just refer to the corresponding name later in the code. But I would suggest to move those "hard-coded" parameters to kedro parameters anyway. Thus I propose resolving this in a separate PR -> created issue #85